ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py

```diff
@@ -148,12 +149,14 @@ def process(self):
         os.rmdir(self.input_file_grp+"/test_A/") #FIXME: better way of deleting a temp_dir?
         
 
-    def _process_segment(self, model, dataset, page, page_xywh, page_id, input_file, n):
+    def _process_segment(self, model, dataset, page, page_xywh, page_id, input_file, orig_img_size, n):
         for i, data in enumerate(dataset):
+            w,h = orig_img_size
             generated = model.inference(data['label'], data['inst'], data['image'])
             dewarped = array(generated.data[0].permute(1,2,0).detach().cpu())
```

Comment:
You mean in the sense, that each output pixel would be mapped to an input pixel so you can reverse engineer a transformation matrix?

Reply:
Exactly. We are already [discussing](https://github.com/OCR-D/core/pull/371#issuecomment-564460874) how to ensure coordinate consistency with dewarping, and your processor would probably become the first example.

Reply:
we discussed this internally and we didnt see a feasible way of aligning pixel before and after the dewarping. following the information flow through the network would lead to a distribution of pixel that are all partially influenced (so kind of a prob. distribution) and more traditional alignment methods would highly depend on the shape (binarization = no color information) and therefore the level of warping in the original image

Reply:
I see. Too bad.